### PR TITLE
Update look of the example app

### PR DESCRIPTION
**Pull request by danny-stytch**

### CODE REVIEW

**Review by mary-stytch — COMMENTED** [review]
Please make the same changes from the html example app here.

File: src/Login.js [hunk 1]
```diff
@@ -16,8 +16,14 @@ const Login = () => {
     },
     style: {
       fontFamily: '"Helvetica New", Helvetica, sans-serif',
-      button: { backgroundColor: "#106ee9" },
-      input: { textColor: "#090909" },
+      borderRadius: 500,
```
Comment (mary-stytch) [inline src/Login.js:19]: I think this should be `borderRadius: 10` or something else much smaller.

File: src/App.css [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+  background: #0577CA;
+  border-radius: 5px;
+  color: #FFFFFF;
+  font-size: 18px;
+  font-weight: Bold;
+  left: 0;
+  line-height: 49px;
+  margin-left: Auto;
+  margin-right: Auto;
+  position: Absolute;
```
Comment (mary-stytch) [inline src/App.css:29]: stylenit: please use lowercase for bold and absolute

File: src/App.css [hunk 3]
```diff
@@ -1,5 +1,39 @@
+body {
```
Comment (mary-stytch) [inline src/App.css:1]: I'm wondering why you need to apply styles to both the `body` element and the `.App` element. If you apply the background color here to the `.App` element, can we delete the CSS for `body`?

Comment (danny-stytch) [reply src/App.css]: Yeah, I didn't need to. Just updated it!

File: src/App.css (hunk 2 above)
Comment (danny-stytch) [reply src/App.css]: Woops, my bad.

File: src/Login.js (hunk 1 above)
Comment (danny-stytch) [reply src/Login.js]: Done

File: src/App.css [hunk 4]
```diff
@@ -1,5 +1,35 @@
+
```
Comment (mary-stytch) [inline src/App.css:1]:
```suggestion
```

**Review by mary-stytch — APPROVED** [review]
One small nit. LGTM!

**Pull request merged**